oommfc/scripts/driver.py

```diff
@@ -29,12 +29,11 @@ def driver_script(
         mif += oc.scripts.evolver_script(driver.evolver)
 
         # Oxs_UZeeman
-        Hmin, Hmax, n = kwargs["Hmin"], kwargs["Hmax"], kwargs["n"]
         mif += "# OxS_UZeeman\n"
         mif += "Specify Oxs_UZeeman:hysteresis {\n"
         mif += "  Hrange {\n"
-        mif += "    {{ {} {} {} {} {} {} {} }}\n".format(*Hmin, *Hmax, n - 1)
-        mif += "    {{ {} {} {} {} {} {} {} }}\n".format(*Hmax, *Hmin, n - 1)
+        for Hstart, Hend, n in kwargs["Hsteps"]:
```

Comment:
We assume here, that only "Hsteps" is existing in kwargs (got e.g. constructed out of Hmin, Hmax and n in the _checkargs() method in oommfc/drivers/hysteresisdriver.py). For that, kwargs needs to be given back through the whole call stack till this point.